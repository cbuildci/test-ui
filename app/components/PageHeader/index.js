/*
 * PageHeader
 */

import React from 'react';
import PropTypes from 'prop-types';

function PageHeader({ children }) {
    return (
        <h1 className="mt-3 pb-2" style={{ paddingBottom: '8px', borderBottom: '1px solid #CCC' }}>
            {children}
        </h1>
    );
}

PageHeader.propTypes = {
    children: PropTypes.element.isRequired,
};

export default PageHeader;
