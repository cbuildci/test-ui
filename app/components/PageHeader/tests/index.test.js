import React from 'react';
import { snapshots } from '../../../../internals/testing/snapshot-util';
import PageHeader from '../index';

describe('<PageHeader />', () => {
    it('should render expected JSX', () => {
        snapshots(
            <PageHeader>
                Foobar
            </PageHeader>,
        );
    });
});
