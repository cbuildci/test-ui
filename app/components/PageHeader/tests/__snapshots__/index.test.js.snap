// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> should render expected JSX 1`] = `
<h1
  className="mt-3 pb-2"
  style={
    Object {
      "borderBottom": "1px solid #CCC",
      "paddingBottom": "8px",
    }
  }
>
  Foobar
</h1>
`;

exports[`<PageHeader /> should render expected JSX 2`] = `
<h1
  class="mt-3 pb-2"
  style="padding-bottom:8px;border-bottom:1px solid #CCC"
>
  Foobar
</h1>
`;
